apps/schedules/lib/by_stop/schedules_by_stop_repo.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Schedules.ByStop.SchedulesByStopRepo do
+  @moduledoc "Repo for getting departure information about all the routes at a given stop"
+
+  alias Schedules.ByStop.DeparturesByStop
```

Comment:
The naming feels sorta redundant to me, maybe just `... .ByStop.Schedules` and `... .ByStop.Departures` would suffice?